src/jit/emit.cpp

```diff
@@ -3341,23 +3341,23 @@ void emitter::emitDispIGlist(bool verbose)
 void emitter::emitDispGCinfo()
 {
     printf("Emitter GC tracking info:");
-    printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
+    // printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
     dumpConvertedVarSet(emitComp, emitPrevGCrefVars);
```

Comment:
I don't understand: you can't delete the code that prints the name of the varset being displayed.